java/ql/src/Security/CWE/CWE-1204/StaticInitializationVector.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Using a static initialization vector for encryption
+ * @description An initialization vector (IV) used for ciphers of certain modes (such as CBC or GCM) should be unique and unpredicateble.
```

Comment:
The description is a bit long and I am not sure of the use of "unpredicateble" (I first read it as `unpredictable` - is this what you meant?) - It would be great if we had a simplified description 🤞🏻 

What about:
An initialization vector (IV) used for ciphers of certain modes (such as CBC or GCM) should be unique and unpredictable (or whatever was the term you meant) to maximize encryption and prevent dictionary attacks.

Reply:
I did mean `unpredictable`, thanks.